src/attachment/test/files.test.ts

```diff
@@ -0,0 +1,59 @@
+import { expect, fixture, elementUpdated } from "@open-wc/testing";
+import { Attachment } from "../index";
+
+const files = [
+  { name: "file.txt", size: "150" },
+  { name: "icon.jpg", size: "200" },
+];
+const invalidFileSizeFiles = [
+  { name: "file.txt", size: "text" },
+  { name: "icon.jpg", size: "200" },
+];
+describe("Attachment", () => {
+  describe("value", () => {
+    it("should be none checked items when not assinged on constructor", async () => {
+      const container = new Attachment({ files: files });
+      const el = await fixture(container);
+      const lisEl = el.querySelectorAll(
+        ".kuc-attachment__group__files__display-area__item__name"
+      );
+      expect(lisEl.length).to.equal(files.length);
+      lisEl.forEach((liEl, index) => {
```

Comment:
please don't write logic in unit test.
we will expect directly
expect((lisEl[0] as HTMLDivElement).innerText).to.equal(files[0].name);
expect((lisEl[1] as HTMLDivElement).innerText).to.equal(files[1].name);